demo/lib/vm.bash

```diff
@@ -15,11 +15,40 @@ VM_GOVM_COMPOSE_TEMPLATE="vms:
 "
 
 vm-check-env() {
+    type -p govm >& /dev/null || {
+        echo "ERROR:"
+        echo "ERROR: environment check failed:"
+        echo "ERROR:   govm binary not found."
+        echo "ERROR:"
+        echo "ERROR: You can install it using the following commands:"
+        echo "ERROR:"
+        echo "ERROR:     git clone https://github.com/govm-project/govm"
+        echo "ERROR:     cd govm"
+        echo "ERROR:     go build -o govm"
+        echo "ERROR:     cp -v govm \$GOPATH/bin"
+        echo "ERROR:     cd .."
+        echo "ERROR:"
+        return 1
+    }
+    docker inspect govm/govm >& /dev/null || {
+        echo "ERROR:"
+        echo "ERROR: environment check failed:"
+        echo "ERROR:   govm/govm docker image not present (but govm needs it)."
+        echo "ERROR:"
+        echo "ERROR: You can install it using the following command:"
+        echo "ERROR:"
+        echo "ERROR:     docker pull govm/govm"
```

Comment:
It seems this pulls govm image with Qemu version earlier than 5.0.

Perhaps we should recommend creating govm image with a Qemu that supports `dies`:

```
docker build go/src/github.com/govm-project/govm -t govm/govm:latest
```

Reply:
Ok, I'll change this so that we'll build it locally together with govm. That's the safest anyway to keep them in sync.

Reply:
Updated printed errors/instructions to build latest docker image from a freshly cloned repo.